File: companyPython/Vendedor.py
from Funcionario import Funcionario

class Vendedor(Funcionario):
    def __init__(self, nome, cpf, salarioBase, comissao):
        super().__init__(nome, cpf, salarioBase)
        self.comissao = comissao

    def calculaSalario(self):
        return self.salarioBase + self.comissao

File: companyJava/Vendedor.java
package empresa;

public class Vendedor extends Funcionario{
	double comissao;
	
	public Vendedor (String nome, String cpf, double salario, double comissao) {
		super(nome, cpf, salario);
		this.comissao = comissao;
	}
	
	@Override
	protected double calculaSalario() {
		return this.salarioBase + this.comissao;
	}
}